src/main/java/com/thealgorithms/others/cn/HammingDistance.java

```
@@ -0,0 +1,41 @@
+package com.thealgorithms.others.cn;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HammingDistance {
+
+
+    private List<byte[]> extractBinaryData(String senderBits, String receiverBits){
+        List<byte[]> byteArray = new ArrayList<>();
+
+        byteArray.add(senderBits.getBytes());
+        byteArray.add(receiverBits.getBytes());
+
+        return byteArray;
+    }
+
+    public int getHammingDistanceBetweenBits(String senderBits, String receiverBits) {
+
+        if(senderBits.length() != receiverBits.length()) {
+           return -1;
```

Please raise an exception instead

---

Yes, reverted the change